### ISSUE TITLE
Questions about the training.

### ISSUE THREAD

**Issue by huagl** [Nanne/pytorch-NetVlad#6]
Hi, Nanne. You have done a great jod on reproducing Net-Vlad!! I have a little problem on traning this network by using your code. I used almost all the default parameters as you wrote in Readme.md, 
`python main.py --mode=train --arch=vgg16 --pooling=netvlad --num_clusters=64 --batchSize=2`
However, I found that the network seems didn't learn anything. Here are my training result,
![image](https://user-images.githubusercontent.com/36216688/56087970-0c476400-5ea9-11e9-90a8-589f6be18a7e.png)
![image](https://user-images.githubusercontent.com/36216688/56087964-ec17a500-5ea8-11e9-9d99-2f4bf432e9fd.png)
Is there anything wrong with me? And how do you set these training parameter to get the performance that you wrote in Readme.md?

**Comment by Nanne** [comment]
That is surprising, although in general I've found Netvlad quite fickle w.r.t. training, but in this case it might be the smaller batchsize. Could you try with a lower learning rate?

**Comment by huagl** [comment]
OK, I will have a try. Did you record these hyper-parameters that got the best performance? It takes a lot of time to train this network☹.

**Comment by Nanne** [comment]
Default hyper-parameters are the ones used to get the scores reported in the README, I didn't spend a lot of time optimising them, so you might be able to find better ones. 

The slowness seems kinda inherent to the approach I'm afraid, I've tried to optimise it a bit, but this is as far as I got in the time that I had for this project.

**Comment by huagl** [comment]
All right, I will do the experiments to optimise them.
 
Anyway, what you have done to reproduce Net-vlad is great, thanks for your work again!
